ledger-tool/src/blockstore.rs

```diff
@@ -328,11 +328,11 @@ pub fn blockstore_subcommands<'a, 'b>(hidden: bool) -> Vec<App<'a, 'b>> {
             .arg(&starting_slot_arg)
             .arg(&ending_slot_arg)
             .arg(
-                Arg::with_name("target_db")
-                    .long("target-db")
+                Arg::with_name("target_ledger")
+                    .long("target-ledger")
                     .value_name("DIR")
                     .takes_value(true)
-                    .help("Target db"),
+                    .help("Target ledger directory to write inner \"rocksdb\" within."),
```

Comment:
I see a handful of other commands that could use some attention. I might try to touch those up in a subsequent PR. But, this is definitely better than what was there so LGTM